Kitodo/src/test/java/org/kitodo/production/services/data/ProcessServiceIT.java

```diff
@@ -162,19 +162,19 @@ public void shouldFindByMetadata() throws DataException {
     }
 
     @Test
-    public void shouldFindByMetadataContent() throws DataException {
-        assertEquals(processNotFound, 1, processService.findByMetadataContent("Proc").size());
+    public void shouldNotFindByMetadata() throws DataException {
+        assertEquals("Process was found in index!", 0,
+                processService.findByMetadata(Collections.singletonMap("TSL_ATS", "Nope")).size());
     }
 
     @Test
-    public void shouldNotFindByMetadataContent() throws DataException {
-        assertEquals(processNotFound, 0, processService.findByMetadataContent("Nope").size());
+    public void shouldFindByMetadataContent() throws DataException {
+        assertEquals(processNotFound, 1, processService.findByAnything("SecondMetaShort").size());
```

Comment:
Will it really test metadata? Maybe test should be called differently if it calls findByAnything?

Reply:
Here it actually tests to find a Process by Metadatacontent